AppKit/CPImage.j

```diff
@@ -85,7 +88,7 @@ function CPImageInBundle()
 
     if (typeof(arguments[1]) === "number")
     {
-        if (arguments[1] != nil)
+        if (arguments[1] !== nil && arguments[1] !== undefined)
```

Comment:
Please use `arguments[1] != nil` instead of `arguments[1] !== nil && arguments[1] !== undefined` as it is equivalent and much more clean looking

Reply:
And I guess we could remove this if statement altogether as `typeof(arguments[1]) === "number")`  will imply that arguments[1]` can't be `nil` or `undefined`.